src/Application/Characters/Models/CharacterCompetitiveRatingViewModel.cs

```diff
@@ -0,0 +1,14 @@
+﻿using AutoMapper;
+using Crpg.Application.Common.Interfaces;
+using Crpg.Application.Common.Mappings;
+using Crpg.Application.Common.Services;
+using Crpg.Application.Parties.Models;
+using Crpg.Domain.Entities.Characters;
+using Crpg.Domain.Entities.Parties;
+
+namespace Crpg.Application.Characters.Models;
+
+public record CharacterCompetitiveRatingViewModel
+{
+    public double CompetitiveRating { get; init; }
+}
```

Comment:
You could move that into the existing rating object. Also I just `float` are being used.

Reply:
same problem : 
i did not want it to be computed in the mod as when the competitive rating formula would change we would get rating from different version mixed in the leaderboard